.github/workflows/pull_request.yml

```diff
@@ -30,26 +30,26 @@ jobs:
         run: pip install pipenv==2018.11.26
       - name: Cache virtualenv
         id: cache-virtualenv
-        uses: actions/cache@v1
+        uses: actions/cache@v3
         with:
           path: ~/.local/share/virtualenvs/
           key: ${{ runner.os }}-${{ env.PYTHON_VERSION }}-virtualenvs-${{ hashFiles('Pipfile.lock') }}
       - name: Install virtual environment
         if: steps.cache-virtualenv.outputs.cache-hit != 'true'
         run: pipenv install --dev
   node-dependencies:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-20.04
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v1
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
         with:
-          node-version: "14.19.0"
+          node-version: "14"
```

Comment:
Hi, Just wanted to check
The earlier version had specific version(14.9.0), here we have changed it to Major Version (14). Any specific reason for this change?

Reply:
Hi, I have refactored this back to version 14.19.0. Cheers.